client/ayon_maya/plugins/publish/save_scene.py

```diff
@@ -12,6 +12,7 @@ class SaveCurrentScene(plugin.MayaContextPlugin):
     label = "Save current file"
     order = pyblish.api.ExtractorOrder - 0.49
     families = ["renderlayer", "workfile"]
+    targets = ["local", "remote"]
```

Comment:
This seems to do the **opposite** of what we want to achieve... now the farm job will ALSO trigger this - but we don't want it to? We ONLY want to trigger this as we submit it to the farm (the UI publish session from the artist).

I believe this should only target `local`, right?